ludwig/data/preprocessing.py

```diff
@@ -1305,6 +1305,12 @@ def build_preprocessing_parameters(
     return feature_name_to_preprocessing_parameters
 
 
+def is_input_feature(feature_config: FeatureConfigDict) -> bool:
+    """Utility function to check for the presence of encoder in the feature config to determine if the feature is
+    an input feature or output feature."""
+    return ENCODER in feature_config
```

Comment:
Pretty minor but could you add a test for this? Could come in handy in case we change what's (not) in the config

Reply:
Done!